src/conv-core/cpuaffinity.C

```diff
@@ -1097,6 +1036,13 @@ void CmiInitCPUAffinityUtil(void){
     }
 */
 #endif
+
+  cmi_hwloc_topology_init(&topology);
+  cmi_hwloc_topology_load(topology);
+
+  cmi_hwloc_topology_init(&legacy_topology);
+  cmi_hwloc_topology_set_flags(legacy_topology, cmi_hwloc_topology_get_flags(legacy_topology) | HWLOC_TOPOLOGY_FLAG_INCLUDE_DISALLOWED);
```

Comment:
the flag was `HWLOC_TOPOLOGY_FLAG_WHOLE_SYSTEM` before. is this intentional?

Reply:
Yes, the `HWLOC_TOPOLOGY_FLAG_WHOLE_SYSTEM` was deprecated and renamed in hwloc 2.1.0.